rpm/0004-Make-funwind-tables-the-default-for-all-archs.patch

```diff
@@ -22,18 +22,16 @@ index 18400d9def5..2643e9decc3 100644
  
  Tool *ToolChain::getClang() const {
 diff --git a/clang/lib/Driver/ToolChains/Gnu.cpp b/clang/lib/Driver/ToolChains/Gnu.cpp
-index e8ef881e89a..088a21455ac 100644
+index e7f57c8249615c65869f1b652da3a7d427f32808..9c44b2e741ce69831f4a892b48eae8ee3073535d 100644
 --- a/clang/lib/Driver/ToolChains/Gnu.cpp
 +++ b/clang/lib/Driver/ToolChains/Gnu.cpp
-@@ -2610,7 +2610,7 @@ void Generic_GCC::printVerboseInfo(raw_ostream &OS) const {
+@@ -2692,6 +2692,9 @@ void Generic_GCC::printVerboseInfo(raw_ostream &OS) const {
  }
  
  bool Generic_GCC::IsUnwindTablesDefault(const ArgList &Args) const {
--  return getArch() == llvm::Triple::x86_64;
-+  return true;
- }
- 
- bool Generic_GCC::isPICDefault() const {
--- 
-2.20.1
-
++  if (getTriple().isMeeGo()) {
++    return true;
++  }
+   switch (getArch()) {
+   case llvm::Triple::aarch64:
+   case llvm::Triple::ppc:
```

Comment:
This was the most interesting conflict of the rebase. I'm not sure whether this still takes the commit message to heart .This might need some change in either the commit message or in the code (to replace it again with `return true;`)